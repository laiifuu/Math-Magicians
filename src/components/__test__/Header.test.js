import Header from '../Header'
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

describe("Tests Header Component", () => {
    it('should render Header component', () => {
        const tree = renderer.create(<BrowserRouter ><Header /></BrowserRouter>).toJSON();
        expect(tree).toMatchSnapshot();
    });
})