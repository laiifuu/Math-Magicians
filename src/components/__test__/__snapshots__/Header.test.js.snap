// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Header Component should render Header component 1`] = `
<header>
  <h1>
    Math Magicians
  </h1>
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</header>
`;
